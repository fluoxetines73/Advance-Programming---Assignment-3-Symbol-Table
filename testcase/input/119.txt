INSERT e_123_2r string
BEGIN
ASSIGN e_123_2r 'HelloWorld'
BEGIN
INSERT 2_3f 123
INSERT g_1 number
BEGIN
END
END
END