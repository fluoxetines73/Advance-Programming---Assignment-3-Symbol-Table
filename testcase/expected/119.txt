Invalid: INSERT 2_3f 123